.github/workflows/build.yml

```diff
@@ -87,6 +87,26 @@ jobs:
         name: Test results (${{ matrix.framework }})
         path: ./test-results
 
+    - name: Nuget Packaging
+      id: nuget
+      run: dotnet pack --include-symbols --no-restore --no-build -p:SymbolPackageFormat=snupkg --property:Platform="Any CPU" --output ./artifacts
+      if: matrix.framework == 'net8.0-windows'
+
+    - name: Upload Nuget Packages
+      id: nuget-temp
+      uses: actions/upload-artifact@v4
+      with:
+        name: Nuget Packages
+        path: |
+          ./artifacts/*.nupkg
+          ./artifacts/*.snupkg
+      if: github.event_name == 'pull_request' && matrix.framework == 'net8.0-windows'
```

Comment:
is there a good reason not to also upload release packages as artifacts?

Reply:
Arbitrary decision since I couldn't immediately think of a reason to use them as they would be published to Nuget. It's a 'Why keep something I don't know I need.' vs 'It doesn't cost anything but a bit of build time to keep it'